app/views/tags/index.html.erb

```diff
@@ -0,0 +1,129 @@
+<%= stylesheet_link_tag "/lib/leaflet/dist/leaflet.css" %>
+<%= stylesheet_link_tag "/lib/leaflet-environmental-layers/dist/LeafletEnvironmentalLayers.css" %>
+<%= javascript_include_tag('/lib/leaflet-environmental-layers/dist/LeafletEnvironmentalLayers.js') %>
+<%= javascript_include_tag('/lib/leaflet-environmental-layers/dist/windRoseLayer.js') %>
+<%= javascript_include_tag('/lib/leaflet-fullhash/leaflet-fullHash.js') %>
+<%= javascript_include_tag('https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/Leaflet.fullscreen.min.js') %>
+<%= stylesheet_link_tag "https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/leaflet.fullscreen.css" %>
+
+<%= javascript_include_tag('/lib/leaflet-spin/example/spin/dist/spin.min.js') %>
+<%= javascript_include_tag('/lib/leaflet-spin/example/leaflet.spin.min.js') %>
+
+<div class="container-fluid">
+  <div id="mapknitter-unique" class="mx-auto" style="height: 450px; width: 100%;">
+  </div>
+  <div class="galary-template">
+    <%= render :template => 'front_ui/gallery' %>
```

Comment:
@divyabaid16 My suggestion is here we  render the maps partial like so
`  <%= render :partial => 'front_ui/maps', :locals => { :maps => @maps } %>`
So we don't include the other tabs in gallery i.e `All Maps`  `anonymous` etc